package.json

```diff
@@ -21,7 +21,8 @@
   "license": "Apache-2.0",
   "private": true,
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "examples/*"
```

Comment:
I was thinking to do it without adding `examples` to workspace. But if this works well then it's OK